Dockerfile.ci

```diff
@@ -148,13 +148,8 @@ RUN mkdir -pv /usr/share/man/man1 \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-# Install Docker
-RUN curl --fail --location https://download.docker.com/linux/debian/gpg | apt-key add - \
-    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian stretch stable" \
-    && apt-get update \
-    && apt-get -y install --no-install-recommends docker-ce \
-    && apt-get autoremove -yqq --purge \
-    && apt-get clean && rm -rf /var/lib/apt/lists/*
+RUN curl https://get.docker.com/builds/Linux/i386/docker-latest -o /usr/bin/docker \
```

Comment:
This is just the `docker` client binary, right? :+1:

Reply:
i386 not x64_86?

Reply:
Oh what

```
❯ http https://get.docker.com/builds/Linux/i386/docker-latest
HTTP/1.1 200 OK
...
```

but

```
❯ http https://get.docker.com/builds/Linux/x86_64/docker-latest
HTTP/1.1 403 Forbidden
...
```

What.

Reply:
That is  a very old docker image. Experimenting still/.

Reply:
This seems to be best:

```
ARG DOCKER_CLI_VERSION=19.03.9
ENV DOCKER_CLI_VERSION=${DOCKER_CLI_VERSION}

RUN curl https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_CLI_VERSION}.tgz \
    |  tar -C /usr/bin --strip-components=1 -xvzf - docker/docker
```